pkg/subctl/cmd/validate_cni.go

```diff
@@ -207,7 +218,7 @@ func validateCalicoIPPoolsIfCalicoCNI(config *rest.Config) {
 		}
 	}
 
-	status.End(status.ResultFromMessages())
```

Comment:
Should we really remove this status.End? or keep it (in addition to the return true?)

Reply:
We need to keep this and also return false if there were any errors. 

```
result := status.ResultFromMessages()
status.End(result)

return result != cli.Failure
```